packages/angular/cli/lib/config/schema.json

```diff
@@ -322,7 +322,7 @@
             "library"
           ]
         },
-        "architect": {
+        "targets": {
```

Comment:
either one (but not both at the same time) should be allowed for 7.x.  Unfortunately, this makes the schema far more complicated.